### ISSUE TITLE
Collaboration Cycle for [Team Name, Product Name, Feature Name]

### ISSUE THREAD

**Issue by mnorthuis** [department-of-veterans-affairs/va.gov-team#68202]
### Verify information about your product or feature in the VFS Product Directory.

- [X] I have reviewed the [VFS Product Directory](https://depo-platform-documentation.scrollhelp.site/getting-started/vfs-product-directory) and confirm that my information is correct (or that I have taken appropriate actions to update or add my product information to the directory).

### VFS team name

Sitewide Content, Accessibility, and IA

### Product name

Benefit hubs

### Feature name

Family member benefit hub

### GitHub label for product

n/a

### Public DSVA Slack channel for VFS team

#sitewide-content-accessibility-ia

### When did/will you start working on this product/feature?

August 2023

### Will your work result in visible changes to the user experience?

Yes

### Are you doing research with VA.gov users?

Yes

### Will your work involve changes to...

Static pages

### Does your product/feature have Google Analytics tracking and a KPI dashboard in Domo?

Yes

### Do you need to capture any additional analytics or metrics?

Yes

### Product outline

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/content-strategy-ia-collaboration/family-member-hub/initiative-brief.md

### Verify all relevant materials provided to Governance Team

- [X] I have provided all relevant and up-to-date links, screenshots, images, designs, etc. of the as-is version of the product

### Notify the Collaboration Cycle team of this ticket in your team's public Slack channel.

- [X] I acknowledge that I must notify the Governance team by completing the collab cycle kicckoff workflow in my team's Slack channel after submitting this issue. This can be completed by typing "/collab cycle" and selecting the appropriate workflow.

**Comment by shiragoodman** [comment]
@mnorthuis  This Collab Cycle ticket hasn't seen movement in a while. Is this still active? If not, please close.

**Comment by mnorthuis** [comment]
@shiragoodman My apologies, the team did not follow through on checkpoints for this effort - which was all content pages. I could schedule a staging review of it in production or I can close.  Preference?

**Comment by shiragoodman** [comment]
@mnorthuis are the changes live in production?

on a related note, it might be best for us to determine how we might work better to determine if Staging Reviews (or other Collaboration Cycle reviews) are necessary for Content and IA team changes to static pages. We have special processes for design system team. Let me know if you're open to this and we can find time to discuss!

**Comment by mnorthuis** [comment]
Yes, in production. Our original discussions on this effort were that we would do the staging review in production on batches of pages because the links and navigation can't be tested in Drupal preview.  

 Yes, we can discuss static page launches/changes. Most will not go through CC at all, but we can discuss larger efforts like this.
